install.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Install MPD on my Raspberry Pi (System-wide configuration)
+# Joe Hallahan jphinspace@gmail.com
+
+# Display commands as they are run
+set -x
+
+echo "Installing mpd..."
+sudo apt-get install mpd
```

Comment:
Testing comment review thing

Reply:
What happens here if we add -y to apt-get install, maybe can remove a manual step